cni/plugin.go

```diff
@@ -132,6 +137,11 @@ func (plugin *Plugin) Add(args *cniSkel.CmdArgs) error {
 	return nil
 }
 
+// Check is an empty CNI CHECK command handler to ensure all CNI plugins implement CNIAPI.
+func (plugin *Plugin) Check(args *cniSkel.CmdArgs) error {
+	return nil
```

Comment:
Do we not need to implement this in our current CNI plugins? Would you be sending that in a separate PR?

Reply:
We are only upgrading the package version we depend on. We are not yet upgrading the CNI spec version advertised from our CNI plugins, and hence Check API need not be implemented yet. All this does is to ensure the base class to all CNI plugins has an empty stub so that the builds succeed.

Reply:
Got it. Thanks.